src/services/DebtService.sol

```diff
@@ -63,112 +74,88 @@ contract DebtService is AdminService {
-
-        // 3. Supply collateral to Aave
         IPool(AAVE_POOL).supply(C_TOKEN, _cAmt, address(this), 0);
     }
 
-    /**
-     * @notice Increases the collateral amount for this contract's loan with permit, obviating the need for a separate approve tx.
-     *         This function can only be used for ERC-2612-compliant tokens.
-     * @param _cAmt The amount of collateral to be supplied (units: C_DECIMALS).
-     * @param _deadline The expiration timestamp of the permit.
-     * @param _v The V parameter of ERC712 signature for the permit.
-     * @param _r The R parameter of ERC712 signature for the permit.
-     * @param _s The S parameter of ERC712 signature for the permit.
-     */
+    /// @notice Increases the collateral amount for this contract's loan with permit, obviating the need for a separate approve tx.
+    ///         This function can only be used for ERC-2612-compliant tokens.
+    /// @dev This function is only callable by the owner account.
+    /// @param _cAmt The amount of collateral to be supplied (units: C_DECIMALS).
+    /// @param _deadline The expiration timestamp of the permit.
+    /// @param _v The V parameter of ERC712 signature for the permit.
+    /// @param _r The R parameter of ERC712 signature for the permit.
+    /// @param _s The S parameter of ERC712 signature for the permit.
     function addCollateralWithPermit(uint256 _cAmt, uint256 _deadline, uint8 _v, bytes32 _r, bytes32 _s)
         public
         payable
         onlyOwner
     {
-        // 1. Approve with permit
         IERC20Permit(C_TOKEN).permit(msg.sender, address(this), _cAmt, _deadline, _v, _r, _s);
-
-        // 2. Add Collateral
         addCollateral(_cAmt);
     }
 
-    /**
-     * @notice Withdraws collateral token from Aave to specified recipient.
-     * @param _token The address of the collateral token to be withdrawn (C_TOKEN or B_TOKEN).
-     * @param _amt The amount of collateral to be withdrawn (units: C_DECIMALS or B_DECIMALS).
-     * @param _recipient The recipient of the funds.
-     */
+    /// @notice Withdraws collateral token from Aave to specified recipient.
+    /// @dev This function is only callable by the owner account.
+    /// @param _token The address of the collateral token to be withdrawn (C_TOKEN or B_TOKEN).
+    /// @param _amt The amount of collateral to be withdrawn (units: C_DECIMALS or B_DECIMALS).
+    /// @param _recipient The recipient of the funds.
     function withdraw(address _token, uint256 _amt, address _recipient) public payable onlyOwner {
         IPool(AAVE_POOL).withdraw(_token, _amt, _recipient);
     }
 
-    /**
-     * @notice Repays any outstanding debt to Aave.
-     * @param _dAmt The amount of debt token to repay to Aave (units: D_DECIMALS).
-     *              To pay off entire debt, _dAmt = debtOwed + smallBuffer (to account for interest).
-     */
+    /// @notice Repays any outstanding debt to Aave.
```

Comment:
`///` is meant for single line comments only.

```solidity
/// @notice Repays outstanding debt to Aave.
/// @dev This function is only callable by the owner account.
/// @param _dAmt The amount of debt token to repay to Aave (units: D_DECIMALS).
/// @dev To pay off entire debt, _dAmt = debtOwed + smallBuffer (to account for interest).
```

Reply:
addressed with @inheritdoc